src/index.ts

```diff
@@ -34,6 +35,10 @@ function checkArtifactExists(artifactsPath: string): void {
  * @returns 0 if succeeded, 1 otherwise
  */
 function processExecuted(executed: ProcessResult): number {
+    if(!executed.stdout){
+        const msg = "Command does not exist. Check if 'cairo-lang' is installed in your active environment.";
```

Comment:
As we said in the call, we aren't sure if this is the only case where `executed.stdout` might be falsy. The message could be framed as:
`Command not found. If you're using a Python virtual environment, check that it has 'cairo-lang' installed.`